packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -78,7 +68,7 @@ export const updateMutation = (t: GraphQLObjectType) => {
 export const deleteMutation = (t: GraphQLObjectType) => {
   const fieldName = getFieldName(t.name, GraphbackOperationType.DELETE)
 
-  return `mutation ${fieldName}($input: ${t.name}Input!) {
+  return `mutation ${fieldName}($input: Delete${t.name}Input!) {
```

Comment:
We have helper for this https://github.com/aerogear/graphback/blob/52cacda4f03dafd5fc1b70a9d2aaf689669fdad5/packages/graphback-core/src/crud/mappingHelpers.ts#L54

Reply:
Ah.. yes. I will see if we can change it